container/src/container_state.rs

```diff
@@ -0,0 +1,187 @@
+use crate::{Error, Result};
+use serde::{Deserialize, Serialize};
+use std::fs;
+use std::fs::{File, OpenOptions};
+use std::io::Write;
+use std::path::PathBuf;
+
+const KAPS_ROOT_PATH: &str = "/var/run/kaps";
```

Comment:
Let that one be `/var/run/kaps/containers/` as we may want to store other, non container specific, pieces of information there.